opt_out_http_api/tests/test_api.py

```diff
@@ -149,12 +211,31 @@ def test_opt_out_count_two_opt_outs(self):
 
     @inlineCallbacks
     def test_opt_out_count_three_opt_outs(self):
-        self.add_opt_out("msisdn", "+271345")
```

Comment:
Delete lines 214-233 and just call `self.backend.put(...)` three times to add three opt outs. Then change the `1` back to a `3` in the expected response.